### ISSUE TITLE
Iphone 5 SE Wheelchair Accessibility Error

### ISSUE THREAD

**Issue by Noznoc** [CSBP-CPSE/CODAP#9]
The wheelchair accessibility's drop-down menu's value is defaulted to "Limited" when there is no actual data for the wheelchair accessibility. The default value should be nothing if there is no existing data regarding the wheelchair accessibility.

**Comment by staubibr** [comment]
I don't see this happening in my browser, it's only for the iPhone?

**Comment by Noznoc** [comment]
Yes, only for iPhone I believe. Jean was using his Android phone and did not have the issue. Just me and Christine, who both have iPhones, had the issue.
